spec/graphql/types/repository_type_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Types::RepositoryType do
+  let(:repository_type) { OntohubBackendSchema.types['Repository'] }
+
+  context 'visibility field' do
+    let(:visibility_field) { repository_type.fields['visibility'] }
+
+    context 'public' do
+      let(:repository) { create :repository, public_access: true }
+      it 'returns public' do
+        visibility = visibility_field.resolve(repository, {}, {})
+        expect(visibility).to eql('public')
```

Comment:
`.to eq(...)`
I think this is the first occurrence of `eql`. These two are aliased.